controllers/mysql_container.go

```diff
@@ -318,19 +342,57 @@ func (r *MySQLClusterReconciler) makeV1InitContainer(cluster *mocov1beta2.MySQLC
 			}),
 	)
 
+	ok, err := r.isEnableLowerCaseTableNames(ctx, cluster)
+	if err != nil {
+		return nil, err
+	}
+	if ok {
+		c.WithArgs(constants.MocoInitLowerCaseTableNamesFlag)
```

Comment:
Please pass the value of `lower_case_table_names` to the moco-init.
Also see https://github.com/cybozu-go/moco-agent/pull/71#discussion_r986465082